scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -228,6 +230,8 @@ class MicrobeEditor{
 
     private void _addOrganelle(PlacedOrganelle@ organelle)
     {
+        if((!organelle.organelle.needNucleus && organelle.organelle.name != "nucleus") || (organelle.organelle.needNucleus && isNucleusPresent) || (organelle.organelle.name == "nucleus" && !isNucleusPresent))
```

Comment:
You might want to turn this condition around and return early as otherwise you'd need to indent this whole function.

Reply:
You have added an IF statement but you didn't indent the function. It's going to start getting pretty tight regards 95 column limit, so I suggest you write that as
```
if(cant place stuff)
    return;
```

instead of indenting the entire function one more level.

Reply:
Ok use the condition that don't allow me to put things instead of good ones.
that are only two, i have nucleus and i'm trying to put it, or i don't have it and i'm trying put something that need it